tutorforum/patches/local-docker-compose-jobs-services

```diff
@@ -2,6 +2,7 @@ forum-job:
   image: {{ FORUM_DOCKER_IMAGE }}
   environment:
     SEARCH_SERVER: "{{ ELASTICSEARCH_SCHEME }}://{{ ELASTICSEARCH_HOST }}:{{ ELASTICSEARCH_PORT }}"
+    {% if ELASTICSEARCH_INDEX_PREFIX|default("") %}ELASTICSEARCH_INDEX_PREFIX: "{{ ELASTICSEARCH_INDEX_PREFIX }}"{% endif %}
```

Comment:
Just a question, I'm not completely sure about it, but:

The `ELASTICSEARCH_INDEX_PREFIX|default("")` is actually meaningless, especially that `default("")` is falsy, ergo it will evaluate to `False` even if the `ELASTICSEARCH_INDEX_PREFIX` is set to `False`, `None`, `0`, or the `default("")` kicks in. 

Following that logic, we could simplify as

```yaml
ELASTICSEARCH_INDEX_PREFIX: "{{ ELASTICSEARCH_INDEX_PREFIX|default("") }}"
```

or am I missing something here?

Reply:
Not really. It's more for cleanliness sake so that the pod's environment doesn't contain an empty key (which looks weird to me).

If you prefer it, I'm happy to change.